src/traces/sunburst/attributes.js

```diff
@@ -215,5 +215,18 @@ module.exports = {
     },
     sort: pieAttrs.sort,
 
+    root: {
+        color: {
+            valType: 'color',
+            editType: 'style',
+            role: 'style',
+            dflt: 'rgba(0,0,0,0)',
+            description: [
+                'sets the color of the root node for a sunburst or a treemap trace.'
```

Comment:
Please mention that this has no effect when `colorscale` is present.

Reply:
addressed in 2f07963